src/Controller/PluginFamily.php

```diff
@@ -388,6 +416,11 @@ public function install_imagify() {
 		}
 
 		$this->set_imagify_partner( '%imagifypartnerid%' );
+		/**
+		 * Fires after Imagify is installed and activated via Plugin Family.
+		 * Allows integrators to track installation/activation.
+		 */
+		do_action( 'wpmedia/plugin_family/imagify_installed' );
```

Comment:
The action naming is not inline with what we use at WP Media

```suggestion
		do_action( 'wpm_pf_imagify_installed' );
```